Solutions/Nasuni/Analytic Rules/RansomwareClientBlocked.yaml

```diff
@@ -0,0 +1,48 @@
+id: 0c96a5a2-d60d-427d-8399-8df7fe8e6536
+name: Ransomware Client Blocked
+description: 'Identifies malicious clients blocked by the Ransomware Protection service running on a Nasuni Edge Appliance.'
+kind: Scheduled
+severity: High
+requiredDataConnectors:
+  - connectorID: NasuniEdgeAppliance
```

Comment:
Hello @mdriscoll-nasuni please correct the spelling of connectorid to ' connectorId ' only I should be capital in the word